### ISSUE TITLE
Home page to get data from a page file?

### ISSUE THREAD

**Issue by orasik** [dataCobra/hugo-vitae#22]
Hi,
Is it possible for the home page to get its content from a file instead of listing posts or pages.?

Example:
```
content/
    posts/
        post1.md
        post2.md
   pages/
       about.md
       home.md
```

I would like home page to show content from `content/pages/home.md`

**Comment by dataCobra** [comment]
Hello @orasik,

currently there is no possibility to change the homepage from a listing to content from a file.
After I read your issue and thought about it, I would really like to add this feature.

I make this issue a feature request and set it as a Milestone for 2.1

Thank you for your issue and the idea. :+1:

**Comment by orasik** [comment]
Thanks Ben!
Glad you liked the idea. I have already implemented it, you can see it here: https://www.30daysofjumpingrope.com (WIP :) )

What I did was changing the `index.html` template and added the page  in `_index.md` file under `content/` folder.

If you like it, I can raise a PR for the enhancement.

Kind regards,
Oras

**Comment by dataCobra** [comment]
Hello @orasik,

I've finally found some time to work on this issue. :+1:

I would really like to see your approach on this feature request.
I'd appreciate it if you could make a pull request. (Please to `2.1-dev` not `master` branch)

**Comment by dataCobra** [comment]
This feature is now available on the 2.1-dev branch.
